expressEngine/model/collection/user.js

```diff
@@ -32,6 +32,12 @@ const UserSchema = new Schema({
     enum: modelLib.roleEnum,
     required: true,
   },
+  firstName: {
+    type: String,
+  },
+  lastName: {
+    type: String,
+  },
```

Comment:
You have included these fields in Schema of User but not initialised those fields in User Service during creating the new user